### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by miradeveloper** [londonappbrewery/Flutter-Course-Resources#52]
Please update your course in Udemy. Am one of your students. Consider the request and do needful

**Comment by vishalrajkumar000** [comment]
yes please bring ad-mod module,local databases,push notifications

**Comment by evanthomas** [comment]
At least answer student questions - I think that is what is expected when you pay money for a course.
